src/assets/toolkit/scripts/lib/component/comment-reply.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+// these two functions should probably be moved to other files
+
+function stringToNode (str) {
+  var div = document.createElement('div');
+  div.innerHTML = str.trim();
+  return div.firstChild;
+}
+
```

Comment:
I tried to find a cleaner way to do this via `document.createDocumentFragment()`, but I failed.